### DISCUSSION TITLE
Question on starter theme and restriction on future updates.

### DISCUSSION THREAD

**Discussion by assayire** [thuliteio/doks#680] [Q&A]
The readme says 

> Use the Doks starter theme, if you plan to customize a lot, and/or do not need future Doks updates.

Pardon my ignorance but why would someone use the starter theme voluntarily signing up NOT to receive future updates? 

I would expect the customization to be on the presentation aspects that may be driven by CSS or custom layouts (picked up from say specific location in the repository). But does that have to prevent from receiving updates (which I suppose should be for the engine that powers doks)?

Appreciate any clarification in this regard. Thanks.

**Comment by h-enk** [comment]
Thanks for bringing this up. I will try to make things clearer in the documentation.

You DO can update the Doks starter theme — it's only more a thing for advanced users (who would know e.g. how to [cherry-pick](https://git-scm.com/docs/git-cherry-pick)).

The Doks child theme uses [Doks as an npm package](https://www.npmjs.com/package/@hyas/doks) — so you can easily update Doks (see also https://github.com/h-enk/doks/discussions/681#discussioncomment-2203648)

You could say, the Doks starter theme is more for advanced users, and the Doks child theme is more for novice to intermediate users.

**Reply by h-enk** [reply]
https://getdoks.org/docs/prologue/quick-start/#create-a-new-site
